src/Events/Services/Interfaces/IInboundService.cs

```diff
@@ -0,0 +1,43 @@
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+using Altinn.Platform.Events.Models;
+
+namespace Altinn.Platform.Events.Services.Interfaces
+{
+    /// <summary>
+    /// Interface to talk to the events service
+    /// </summary>
+    public interface IInboundService
+    {
+        /// <summary>
+        /// Save cloud event to persistent storage.
+        /// </summary>
+        /// <param name="cloudEvent">The cloudEvent to be saved</param>
+        /// <returns>Id for the created document</returns>
+        Task<string> Save(CloudEvent cloudEvent);
+
+        /// <summary>
+        /// Push cloud event to inbound queue.
+        /// Should only be attempted after saving to persistent storage. 
+        /// </summary>
+        /// <param name="cloudEvent">The cloudEvent to be queued</param>
+        /// <returns>Id for the queued event</returns>
+        Task<string> PostInbound(CloudEvent cloudEvent);
+
+        /// <summary>
+        /// [DEPRECATED] Save a cloud event to persistent storage and
```

Comment:
To ensure that the information about the deprecation showing up in build logs etc. you should use `[Obsolete("message")]`
`